CODEOWNERS

```diff
@@ -16,7 +16,7 @@ src/include/user/**			@thesofproject/steering-committee
 src/include/sof/debug/gdb/*		@mrajwa
 src/include/sof/audio/kpb.h		@mrajwa
 src/include/sof/audio/mux.h		@akloniex
-src/include/sof/audio/codec_adapter/* @cujomalainey
+src/include/sof/audio/codec_adapter/* @cujomalainey @mrajwa @dbaluta
```

Comment:
Shouldn't they be added for the source files as well, in `src/audio/codec_adapter/*` ?

Reply:
yes this is just the headers